main/src/library/File.flix

```diff
@@ -15,6 +15,74 @@
  */
 namespace File {
 
+    ///
+    /// Returns `true` if the given file `file` exists.
+    ///
+    pub def exists(file: String): Result[Bool, String] & Impure =
```

Comment:
I would rename the argument from `file` to just `f` and I would do that throughout the code.